superset-frontend/src/views/CRUD/data/dataset/AddDataset/DatasetPanel/DatasetPanel.tsx

```diff
@@ -248,6 +258,9 @@ const DatasetPanel = ({
   const theme = useTheme();
   const hasColumns = columnList?.length > 0 ?? false;
   const datasetNames = datasets?.map(dataset => dataset.table_name);
+  const tablesWithDatasets = datasets?.find(
```

Comment:
should we rename this to singular? `tablesWithDatasets` -> `tableWithDatasets` since `find` only returns one element?